porcupine/settings.py

```diff
@@ -841,7 +841,18 @@ def settings_to_gui(junk: object = None) -> None:
         if panedwindow["orient"] == "horizontal":
             panedwindow.paneconfig(pane, width=global_settings.get(option_name, int))
         else:
-            panedwindow.paneconfig(pane, height=global_settings.get(option_name, int))
```

Comment:
This file isn't the right place for this change. Because we're doing this for the run plugin, the fix belongs to the run plugin. The correct place is probably the `run` method in `class Executor` (file `no_terminal.py`), as it already does stuff with font size and textwidget height.